workflow/signals.py

```diff
@@ -62,6 +62,15 @@ def add_users_to_default_wflvl1(sender, instance, **kwargs):
                                     workflowlevel1=wflvl1_1)
 
 
+@receiver(signals.post_save, sender=Organization)
+def create_default_program(sender, instance, **kwargs):
```

Comment:
You'll create a default program when the organization is updated as well. Check if the `kwargs.get('created')` is `False`.